src/types/DataModels.ts

```diff
@@ -0,0 +1,22 @@
+export class CardContent {
```

Comment:
Same as above. That is recognized as a new file. And it's not the latest version of this file.